.github/workflows/build.yml

```diff
@@ -21,7 +21,10 @@ jobs:
           node-version: 12.x
 
       - run: yarn install --frozen-lockfile
+
       - run: yarn audit
+        continue-on-error: true
```

Comment:
This should not be needed as https://github.com/iTwin/iTwinUI/pull/61 was merged

Reply:
merged